index.js

```diff
@@ -53,11 +58,13 @@ module.exports = {
     app.use(config.nodered.httpAdminRoot, RED.httpAdmin);
     app.use(config.nodered.httpNodeRoot, RED.httpNode);
 
-    if (config.nodered.httpServer)
+    if (config.nodered.httpServer) {
       httpServer.listen(config.rest.port);
+    }
 
-    RED.start();
-
+    RED.start().then(() => {
+      server.emit('serverStart')
```

Comment:
make the init function async and await for Red.start(). What's the point in eventEmitter, if you don't use it?